### DISCUSSION TITLE
Issue resolving registry.quarkus.io/maven for Nexus repo update

### DISCUSSION THREAD

**Discussion by pompom-13** [quarkusio/quarkus#41933] [Q&A]
Hi

we are currently trying to upgrade from Quarkus 3.10.0 to 3.12.1

after installing the latest quarkus-cli (3.12.1) and running

quarkus update

we are getting the following errors (using a corporate Nexus repo)

[WARNING] Failed to resolve the Quarkus extension registry descriptor of registry.quarkus.io from shp.mirror (https://[OUR_NEXUS_REPO].com/repository/SHP-Public/) having applied the mirrors and/or proxies from the Maven settings to registry.quarkus.io (https://registry.quarkus.io/maven). Re-trying with the original registry.quarkus.io repository configuration.
then

[ERROR] Failed to execute goal io.quarkus:quarkus-maven-plugin:3.12.0:update (default-cli) on project [OUR_PROJECT]: Failed to initialize Quarkus extension resolver: Failed to resolve the Quarkus extension registry descriptor of registry.quarkus.io from registry.quarkus.io (https://registry.quarkus.io/maven) -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal io.quarkus:quarkus-maven-plugin:3.12.0:update (default-cli) on project [OUR_PROJECT]: Failed to initialize Quarkus extension resolver

my understanding in this area is rather patchy, so please bear with me.

Using mvn io.quarkus.platform:quarkus-maven-plugin:3.12.1:update -N

I get more or less the same error

I contacted our Nexus server admin with the information from this page https://quarkus.io/guides/extension-registry-user => "How to register as a Nexus Repository proxy"

so that he could add registry.quarkus.io to our Nexus repo

But he doesn't really see how this can work as the Maven url https://registry.quarkus.io/maven returns an HTTP 404

Is this url correct? If so how is it suppose to work in the Nexus config?

Thanks

[a comment by an automated account is not shown]

**Comment by gastaldi** [comment]
> Is this url correct? If so how is it suppose to work in the Nexus config?

Yes, the URL is correct - the repository isn't browseable. The instructions in https://quarkus.io/guides/extension-registry-user#how-to-register-as-a-nexus-repository-proxy should be enough to have it working in Nexus. What Nexus version are you using?

**Reply by pompom-13** [reply]
Hi, thanks for your reply - we're on Nexus 3. I'll ask for the repo to be created as indicated in the doc for Nexus 3.
